_posts/2021-01-14-voicemeeter-banana.md

```diff
@@ -0,0 +1,112 @@
+
+Voicemeter Ba na na, na na na na, sing with me! Unless you are streaming or making a youtube video as that could result in a DMCA! Joke aside, Voicemeeter Banana, what is it? Voicemeeter Banana is the second iteration of Voicemeeter, an audio mixing software that allows advanced audio mixing, which is also provided free. You can download voicemeeter by visiting their website [here](http://voicemeeter.com/).
+
+Voicemeeter Banana comes with three Hardware Inputs, two software o digital inputs, and three hardware outputs. For reference, Hardwar inputs are typically hardware devices pushing audio into the computer, such as microphones, line inputs, and other things like that. The software or digital input is a driver on the system that allows audio to be pushed to it, they help with threading audio around. Hardware outputs are your headphones, speakers and can also be a digital audio driver.
+
+## Installing Voicemeeter Banana
+
+I think the best way to explain Voicemeeter Banana is to set it up and explain along the way. This gives us a more hands on approach which we can test actively. I will cover setting up Voicemeeter in this way: Hardware Outputs, Digital Inputs, then Hardware Inputs. My reason for this is it is hard to test if we have any of the inputs set up if we don't have proper outputs set up. So please install Voicemeeter Banana and the free Virtual Audio Cable that comes with it, you may be required to restart your computer.
+
+Location of Virtual Audio Cable in Voicemeeter after installation:
+'C:\Program Files (x86)\VB\Voicemeeter'
+
+
+## Setting up Voicemeeter Banana
+
+Please open Voicemeeter Banana.
+
+### Saving Settings, Loading Settings, Resetting settings
+Something that typically gets skipped in other Voicemeeter coverage that i've seen is the ability to save settings. At any point you can save your current settings of voicemeeter by going to Menu→Save Settings. This saves a settings file on your computer you can then load up from just a file even on other devices should you want to transfer your settings to another device. If at any point you feel lost and want to restart the setup, you can go to Menu→ Reset Settings (Re-Initialization) and this will make Voicemeeter appear as if you had just installed it, blank with nothing set up.
+
+Image for showing where the menu is
+
+![https://i.imgur.com/bbE1y24.png](https://i.imgur.com/bbE1y24.png)
+
+Image for showing save/load/reset settings
+
+![https://i.imgur.com/bbE1y24.png](https://i.imgur.com/bbE1y24.png)
+
+
+### Audio outputs
+The Hardware Ouputs section in the top right should have A1, A2, A3 with a triangle beside them. Each one of these have a selectable menu for our output devices. Each input device has a toggle-able setting for us to push audio toe each individual output device, which will get  better explanation when we get to inputs inputs. For now, click on A1 and select your headphones. click on A2 and select the virtual audio cable.
```

Comment:
- "toe" -> "to"
- "inputs inputs" -> "inputs"?
- Can we walk them through figuring out what their default device is then reference that here instead of explicitly saying "headphones", or did I miss something?
- Capitalization of "Click" in the final sentence.

Reply:
Maybe it would be a good idea to show them how to change stuff in sound settings -> playback devices instead? Or at least get them there to see what their devices are being called. Windows Default Audio Output Device gets mentioned in the next step, which is the step a lot of people seem to start with in their setup/explanation of voicemeeter. So you didn't really miss anything, I just went in a different order than you might be used to from the other reviews?